model-optimizer/extensions/front/onnx/resize_to_interpolate.py

```diff
@@ -42,7 +42,8 @@ def pattern(self):
                 ('unsqueeze_1', dict(op='ExpandDims')),
                 ('unsqueeze_2', dict(op='ExpandDims')),
                 ('slice', dict(op='Slice')),
-                ('slice_start', dict(op='Const', value=lambda x: x is not None and np.array_equal(x, int64_array([2])))),
+                ('slice_start',
```

Comment:
I think that transformations from this file are not needed anymore. They were developed to workaround issues with resize-ing image to a shape like 11.999999. Now when we support Interpolate-4 this issue should be gone. I think that this transformation  was developed for ONNX Mask-RCNN described here https://docs.openvinotoolkit.org/latest/openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_Mask_RCNN.html. Please, double check that this transformation is triggered for this model and that after removing this transformation the model is still converted and inferred correctly.

Reply:
Done.